library/std/src/process.rs

```diff
@@ -262,6 +262,25 @@ impl Write for ChildStdin {
     }
 }
 
+#[stable(feature = "write_mt", since = "1.47.0")]
+impl Write for &ChildStdin {
```

Comment:
All of these impls duplicate the bodies, but we should have the ownership-taking impl call the `&T` impl instead, to avoid later modifications only changing one and not the other.